docs/bugs_and_glitches.md

```diff
@@ -180,38 +180,7 @@ Fixes in the [multi-player battle engine](#multi-player-battle-engine) category
 
 ### Metal Powder can increase damage taken with boosted (Special) Defense
 
-([Video](https://www.youtube.com/watch?v=rGqu3d3pdok&t=450))
-
-**Fix:** Edit `DittoMetalPowder` in [engine/battle/effect_commands.asm](https://github.com/pret/pokecrystal/blob/master/engine/battle/effect_commands.asm):
-
-```diff
- 	ld a, c
- 	srl a
- 	add c
- 	ld c, a
- 	ret nc
-
- 	srl b
- 	ld a, b
- 	and a
- 	jr nz, .done
- 	inc b
- .done
- 	scf
- 	rr c
-+
-+	ld a, HIGH(MAX_STAT_VALUE)
-+	cp b
-+	jr c, .cap
-+	ret nz
-+	ld a, LOW(MAX_STAT_VALUE)
-+	cp c
-+	ret nc
-+
-+.cap
-+	ld bc, MAX_STAT_VALUE
- 	ret
-```
+_TODO: Document a proper fix._
```

Comment:
```suggestion
([Video](https://www.youtube.com/watch?v=rGqu3d3pdok&t=450))

_TODO: Document a proper fix._
```

(I'd also just leave this open and see if one of us has a fix, first.)

Reply:
If we don't change it more new people will keep following the flawed guide and won't know why it's not working properly.